root/commons/src/main/java/at/fhv/itb5c/commons/property/PropertyManager.java

```diff
@@ -1,16 +1,39 @@
 package at.fhv.itb5c.commons.property;
 
+import java.io.IOException;
 import java.io.InputStream;
+import java.util.Properties;
 
-public class PropertyManager {
+import at.fhv.itb5c.logging.ILogger;
+
+public class PropertyManager implements ILogger {
```

Comment:
maybe [this](http://crunchify.com/java-properties-file-how-to-read-config-properties-values-in-java/) is helpful